src/feature/stats/rephist.c

```diff
@@ -973,47 +971,45 @@ rep_hist_load_mtbf_data(time_t now)
   return r;
 }
 
-/** For how many seconds do we keep track of individual per-second bandwidth
- * totals? */
-#define NUM_SECS_ROLLING_MEASURE 10
-/** How large are the intervals for which we track and report bandwidth use? */
-#define NUM_SECS_BW_SUM_INTERVAL (24*60*60)
-/** How far in the past do we remember and publish bandwidth use? */
-#define NUM_SECS_BW_SUM_IS_VALID (5*24*60*60)
-/** How many bandwidth usage intervals do we remember? (derived) */
-#define NUM_TOTALS (NUM_SECS_BW_SUM_IS_VALID/NUM_SECS_BW_SUM_INTERVAL)
+/** Recent history of bandwidth observations for read operations. */
+static bw_array_t *read_array = NULL;
+/** Recent history of bandwidth observations for write operations. */
+STATIC bw_array_t *write_array = NULL;
```

Comment:
Nit: I don't think this one have to be `STATIC`.

Reply:
It was STATIC from before. Should I change it ??

Reply:
If it's used in the unit tests, say "STATIC", which means "only available in this file in the tor binary, but available to the unit tests".

If it's not used in the unit tests, say "static", which means "only available in this file" (and not available to the unit tests).

Reply:
The variable here is also used in unit tests.  So leaving it as it is.